ui/packages/catalog-ui-search/package.json

```diff
@@ -33,10 +34,10 @@
     "js-beautify": "1.8.7",
     "jsdom": "11.12.0",
     "loader-utils": "1.1.0",
-    "sinon": "5.0.7"
+    "sinon": "5.0.7",
+    "typescript": "3.5.3"
   },
   "dependencies": {
-    "@connexta/atlas": "0.0.35",
```

Comment:
Why remove?

Reply:
We no longer have any dependency on it in DDF.  @blen-desta removed the login usage.